content/gettingstarted/index.adoc

```diff
@@ -16,129 +16,132 @@ toc::[]
 
 [source,bash]
 ----
-bash <(curl -L http://git.io/getLatestKiali)
+minikube addons enable ingress
 ----
 
-By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKiali[the comments at the top of the install script].
+When Kiali is installed in Minikube, this Ingress add-on will enable you to go directly to the Kiali UI at the URL `https://[minikube-ip]/kiali/console` where `[minikube-ip]` is what is reported when you run the command: `minikube ip`.
 
-Once the above has completed and the Docker image has been pulled from Dockerhub, go to the OpenShift console, select the istio-system project and determine the base-URL of Kiali
+=== Install Istio
 
-[#img-openshift]
-.OpenShift console
-image::os-console.png[OpenShift console]
-{nbsp} +
+Make sure your cluster environment has Istio installed. Because you are following these instructions to install the latest Kiali, you should not install the Kiali that comes with Istio (i.e. do not use the Helm setting `kiali.enabled=true` when installing Istio).
 
-In this case it is `http://kiali-istio-system.192.168.64.13.nip.io`. In your case this could be a different IP.
+For details on how to install Istio, see the link:https://istio.io/docs/setup/[Istio Setup docs].
 
-You can also use the `oc` command to determine the base-url:
+=== Kiali Operator
 
-[source,bash]
-----
-oc get route -n istio-system -l app=kiali
-----
+Now install the latest release of Kiali using the Kiali Operator.
 
-== Getting started on Kubernetes
+==== What is the Kiali Operator?
 
-To deploy Kiali to your Kubernetes cluster you can do the following.
+First, a quick introduction of what the Kiali Operator actually is.
 
-=== Preparation
+The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
 
-Preparation is only needed if you are installing in Minikube. When installing in Minikube, ensure that you enable the Ingress add-on by executing the following command:
+The Kiali Operator runs inside the `kiali-operator` namespace and listens for the creation of a Kiali CR (custom resource) object. When it sees one created, it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file]. A minimal Kiali CR that falls back to all defaults looks like this:
```

Comment:
>A minimal Kiali CR that falls back to all defaults looks like this:
A minimal Kiali CR that uses all the defaults looks like this:


(Also yay fully documented example!  \o/)

Reply:
done